apps/browser/lib/supabase/createSupabaseClient.ts

```diff
@@ -16,11 +20,11 @@ export const createSupabaseClient = (supabaseAccessToken: string) => {
     {
       global: {
         headers: {
-          Authorization: `Bearer ${supabaseAccessToken}`,
+          authorization: `Bearer ${supabaseAccessToken}`,
```

Comment:
Shouldn't this be `Authorization`?
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization